### PR TITLE
Upgrade StatsD version

### PR DESCRIPTION
**Pull request by kakkoyun**
* Fix typo
* Update documentation

### CODE REVIEW

**Comment by kakkoyun** [comment]
@salimane Dear sir, I very much appreciate what you have done here. I want to contribute back. Hence this PR fixes a couple of discovered issues and upgrades to the latest version of StatsD.

**Review by alinur-meltwater — APPROVED** [review]

File: statsd/config.js [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
+  graphitePicklePort: port for the graphite pickle collector [default: 2004]
+  graphiteProtocol: either 'text' or 'pickle' [default: 'text']
   backends:         an array of backends to load. Each backend must exist
                     by name in the directory backends/. If not specified,
                     the default graphite backend will be loaded.
                     * example for console and graphite:
                     [ "./backends/console", "./backends/graphite" ]
-  server:           the server to load. The server must exist by name in the directory
+
+  servers:          an array of server configurations.
```
Comment (salimane) [inline statsd/config.js]: could we mention all these changes in the readme ?

**Review by salimane — APPROVED** [review]
🚢

File: statsd/config.js [hunk 2]
```diff
@@ -110,8 +124,8 @@ Optional Variables:
 , debug: (process.env.DEBUG || "false").toLowerCase() == "true"
 , server: "./servers/" + (process.env.SERVER_PROTOCOL || "udp")
 , port: parseInt(process.env.SERVER_PORT) || 8125
-, flushInteval: process.env.FLUSH_INTERVAL || "1000"
-, percentThreshold: [ process.env.PERCENT_TRESHOLD || "90" ]
+, flushInterval: process.env.FLUSH_INTERVAL || "1000"
+, percentThreshold: [ process.env.PERCENT_THRESHOLD || "90" ]
```
Comment (salimane) [inline statsd/config.js:128]: please update the readme to match those changes

File: statsd/config.js [hunk 3]
```diff
@@ -83,8 +97,8 @@ Optional Variables:
                     e.g. [ { host: '10.10.10.10', port: 8125 },
                            { host: 'observer', port: 88125 } ]
 
-  repeaterProtocol: whether to use udp4 or udp6 for repeaters.
-                    ["udp4" or "udp6", default: "udp4"]
+  repeaterProtocol: whether to use udp4, udp6, or tcp for repeaters.
+                    ["udp4," "udp6", or "tcp" default: "udp4"]
```
Comment (salimane) [inline statsd/config.js:101]: it seems you have some typos here

**Review by salimane — CHANGES_REQUESTED** [review]

File: statsd/config.js (hunk 3 above)
Comment (kakkoyun) [reply statsd/config.js:101]: It's direct copy paste from `statsd` repo, I'll double check 🙈

File: statsd/config.js (hunk 2 above)
Comment (kakkoyun) [reply statsd/config.js:128]: Already did dear sir

**Review by salimane — APPROVED** [review]

**Pull request merged**